react-view/src/utils/APIUtils/index.ts

```
@@ -0,0 +1,125 @@
+        .catch((err) => {
+          reject(err);
+        })
+        .finally(() => {});
+    });
+  }
+
+  put(path: string, body: Record<string, any>, options?: RequestInit) {
+    fetch(this.apiHostname.concat(path), {
+      headers: {
+        ...this.commonHeaders,
+        ...(options && options.headers),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)